src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SnakemakeLexer.kt

```diff
@@ -180,11 +185,7 @@ class SnakemakeLexer : PythonIndentingLexer() {
         }
 
         previousToken = tokenType
-        insideFormattedString = if (!insideFormattedString) {
-            tokenType == PyTokenTypes.FSTRING_START
-        } else {
-            tokenType != PyTokenTypes.FSTRING_END
-        }
+        twoPrecedingTokens = twoPrecedingTokens.copy(first = twoPrecedingTokens.second, second = previousToken)
```

Comment:
Is it a sophisticated way to write `twoPrecedingTokens = twoPrecedingTokens.second to previousToken` ?